src/frontend/components/SøknadsSteg/Oppsummering/Oppsummeringsbolk.tsx

```diff
@@ -75,12 +85,16 @@ const Oppsummeringsbolk: React.FC<Props> = ({
                     {steg?.route !== RouteEnum.OmBarnet &&
                         steg?.route !== RouteEnum.EøsForBarn &&
                         `${hentStegNummer(steg?.route ?? RouteEnum.OmDeg)}. `}
-                    <SpråkTekst id={tittel} values={språkValues} />
+                    {tittelForSanity ? (
+                        uppercaseFørsteBokstav(plainTekst(tittelForSanity, flettefelter))
```

Comment:
Hvorfor uppercaser vi første bokstav her?

Reply:
Det er gjort slikt i KS fra før, så jeg gjorde det samme i BA uten å tenke mer på det.

Endringen i KS ble gjort for nesten 2 år siden for å passe på at `ytelse` flette-feltet alltid fikk stor forbokstav. https://github.com/navikt/familie-ks-soknad/pull/134

Jeg finner ingen grunn til å ha den i BA ettersom alle Sanity tekstene som brukes som tittel i `Oppsummeringsbolk` allerede har store bokstaver.

Jeg fjerner bruken av `uppercaseFørsteBokstav` 👍